src/solvers/dgsem_t8code/containers.jl

```diff
@@ -18,27 +18,32 @@ function reinitialize_containers!(mesh::T8codeMesh, equations, dg::DGSEM, cache)
     @unpack boundaries = cache
     resize!(boundaries, mesh.nboundaries)
 
-    trixi_t8_fill_mesh_info(mesh.forest, elements, interfaces, mortars, boundaries,
-                            mesh.boundary_names)
+    fill_mesh_info(mesh, interfaces, mortars, boundaries,
+                   mesh.boundary_names)
```

Comment:
I guess this function modifies the `mesh`? If so, it should be called

```suggestion
    fill_mesh_info!(mesh, interfaces, mortars, boundaries,
                    mesh.boundary_names)
```

Reply:
Good catch! Fixed.